.dockerignore

```diff
@@ -1 +1,12 @@
+.git
+.gitignore
 Dockerfile
+lib
+node_modules
```

Comment:
Why do we want to not put node_modules into the container?

Right now yarn doesn't use it as a cache but it could in the future. See the comment in the Dockerfile.

Reply:
I guess we can add it back later.

Reply:
https://medium.com/@rdsubhas/docker-for-development-common-problems-and-solutions-95b25cae41eb

In linux we have linux binaries, on mac we have mac binaries, on windows we have exe files.

NPM is a hot mess, so node_modules should be git + docker ignored.

Reply:
Are you sure about this? I thought `node_modules` only contained source code? Not actual binaries.

Reply:
@zerdos is right. Check the folder `node_modules/.bin/`. There are modules that build executables in there (the `node-sass` package is one that comes to mind. I don't think code-server uses it, but there are many other examples).